client/state/selectors/get-reader-follows.js

```diff
@@ -15,7 +15,7 @@ import createSelector from 'lib/create-selector';
  * @return {Integer} Follow count
  */
 const getReaderFollows = createSelector(
-	state => values( state.reader.follows.items ),
+	state => values( filter( state.reader.follows.items, item => ! item.error ) ),
```

Comment:
also, does filter work here? `items` is an object, not an array.. should work if you wrap it the other way: `filter( values( ... ) )`